src/CommonProviderImplementation/Helpers.fs

```diff
@@ -147,43 +148,18 @@ module ProviderHelpers =
           SampleIsWebUri : bool
           SampleIsResource : bool }
 
-    type private EmbeddedResourceReader() =
```

Comment:
This AppDomain marshalling was needed to not lock the files, otherwise it was very hard to build FSharp.Data.Toolbox.Twitter. Has something else changed that makes this not needed any more?